cmd/cloudtrail/write-events_test.go

```diff
@@ -0,0 +1,79 @@
+package cloudtrail
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLoadConfiguration(t *testing.T) {
+
+	config, err := LoadConfiguration()
+
+	if err != nil {
+
+		t.Errorf("Error loading configuration: %v", err)
+	}
+
+	if config == nil {
+		t.Error("Configuration is nil")
+	}
+
+}
```

Comment:
**issue:** This test is based on the user's current configuration. We don't want to test this as it could make the result differ for every person running `make test`